packages/tx/src/util/authorization.ts

```diff
@@ -128,7 +128,8 @@ export function signAuthorization(
   privateKey: Uint8Array,
 ): AuthorizationListBytesItem {
   const msgHash = authorizationHashedMessageToSign(input)
-  const signed = ecsign(msgHash, privateKey)
+  // TODO: always uses the JS version: read ecsign from crypto
```

Comment:
```suggestion
  privateKey: Uint8Array,
  common?: Common
): AuthorizationListBytesItem {
  const msgHash = authorizationHashedMessageToSign(input)
  // TODO: always uses the JS version: read ecsign from crypto
```

Should be pretty safe to just add `common` here as an optional param and if it exists, use it.

Reply:
I have added this! :smile: :+1: